src/lib/select/select.ts

```diff
@@ -466,9 +466,11 @@ export class MdSelect extends _MdSelectMixinBase implements AfterContentInit, On
       return;
     }
 
-    if (!this._triggerWidth) {
-      this._setTriggerWidth();
-    }
+    /**
+     * Always set the trigger width since we need to
+     * update the panel with its new minWidth when opening it
+     */
```

Comment:
Nit: For comments inside methods, we typically use `//` comment style. Applies throughout PR.